makerspace/templates/tools/admin_tool.html

```diff
@@ -0,0 +1,90 @@
+{% extends "web/base.html" %}
+{% load staticfiles %}
+{% load i18n %}
+{% load news_tags %}
+
+{% block head %}
+    <link rel="stylesheet" type="text/css" href="{% static "news/css/admin.css" %}">
+{% endblock head %}
+
+{% block body %}
+    <h1 class="ui text-centered make_col_yellow header">
+        {% trans "Tools" %}
+        <a class="make_col_blue explanation-popup" data-title="{% trans "Create" %}"
+           data-content="{% trans "If clicked, a new Tool will be created." %}" href="{% url 'makerspace-tools-create' %}">
+            <i class="ui add icon"></i>
+        </a>
+    </h1>
+    <div class="ui text container">
+        {% if tools_list %}
+            <div class="ui divided list">
+                {% for tool in tools_list %}
+                    <div class="item">
+                        <i class="ui icon"><img class="inline square_36" src="{{ tool.image.url }}"></i>
```

Comment:
When I created a tool without an image and was redirected to the tools admin page this fails. I get an `ValueError: The 'image' attribute has no file associated with it.`.  Either check if a tool has an image before trying to show it, or require all tools to have an image in the tool form.